.gitignore

```diff
@@ -108,3 +108,6 @@ venv.bak/
 
 # sqlite
 *.db
+
```

Comment:
If we are dropping support for sqlite then we can probably remove these lines from the `.gitignore`.